.github/workflows/r_package.yml

```diff
@@ -8,6 +8,18 @@ on:
     branches:
     - master
 
+env:
+  # hack to get around this:
+  # https://stat.ethz.ch/pipermail/r-package-devel/2020q3/005930.html
+  _R_CHECK_SYSTEM_CLOCK_: 0
+  # ignore R CMD CHECK NOTE checking how long it has
+  # been since the last submission
+  _R_CHECK_CRAN_INCOMING_REMOTE_: 0
+  # CRAN ignores the "installed size is too large" NOTE,
+  # so our CI can too. Setting to a large value here just
+  # to catch extreme problems
+  _R_CHECK_PKG_SIZES_THRESHOLD_: 100
```

Comment:
I believe we can now drop these lines

https://github.com/microsoft/LightGBM/blob/9e1d7fa1bb395a9c4e5f7af9b13b9eb315e360fa/.ci/test_r_package_windows.ps1#L82-L93

Reply:
oops yes! Forgot about the powershell script, thank you

Reply:
removed in https://github.com/microsoft/LightGBM/pull/4164/commits/7f8547a7d361e5bbc5da6f032c8919282a86b5da